src/plugins/orangehrmDashboardPlugin/Dto/EmployeeOnLeaveSearchFilterParams.php

```diff
@@ -49,4 +57,36 @@ public function setDate(DateTime $dateTime): void
     {
         $this->dateTime = $dateTime;
     }
+
+    /**
+     * @return DateTime
+     */
+    public function getDateTime(): DateTime
+    {
+        return $this->dateTime;
+    }
+
+    /**
+     * @param DateTime $dateTime
+     */
+    public function setDateTime(DateTime $dateTime): void
+    {
+        $this->dateTime = $dateTime;
+    }
```

Comment:
```suggestion
```
remove these getter and setter, `getDate`, `setDate` already there

Reply:
Oops, this was my mistake when generating the getters and setters